src/test/features/response/routes/partial-admission/how-much-have-you-paid.ts

```diff
@@ -4,29 +4,27 @@ import * as config from 'config'
 
 import { attachDefaultHooks } from 'test/routes/hooks'
 import 'test/routes/expectations'
-import { checkAuthorizationGuards } from 'test/features/response/routes/checks/authorization-check'
-import { checkAlreadySubmittedGuard } from 'test/features/response/routes/checks/already-submitted-check'
 
-import { Paths as ResponsePaths } from 'response/paths'
-import { ResponseType } from 'response/form/models/responseType'
+import { Paths, PartAdmissionPaths } from 'response/paths'
 
 import { app } from 'main/app'
 
 import * as idamServiceMock from 'test/http-mocks/idam'
-import * as draftStoreServiceMock from 'test/http-mocks/draft-store'
 import * as claimStoreServiceMock from 'test/http-mocks/claim-store'
+import * as draftStoreServiceMock from 'test/http-mocks/draft-store'
+import { checkAuthorizationGuards } from 'test/features/response/routes/checks/authorization-check'
+import { ResponseType } from 'response/form/models/responseType'
 import { checkNotDefendantInCaseGuard } from 'test/features/response/routes/checks/not-defendant-in-case-check'
 
 const cookieName: string = config.get<string>('session.cookieName')
-const pagePath = ResponsePaths.defenceRejectPartOfClaimPage.evaluateUri({ externalId: claimStoreServiceMock.sampleClaimObj.externalId })
+const externalId = claimStoreServiceMock.sampleClaimObj.externalId
+const pagePath = PartAdmissionPaths.howMuchHaveYouPaidPage.evaluateUri({ externalId: externalId })
+
+const validFormData = { amount: 100, date: { day: 1, month: 1, year: 1990 }, text: 'aaa' }
+const header: string = 'How much have you paid the claimant?'
```

Comment:
Out of interest - when the 'claimant' is actually multiple people are we going to have to revisit these, or will they be totally separate journeys?

Reply:
all will be revisited